packages/ui/src/components/Modal/components/Dialog.tsx

```diff
@@ -111,9 +123,13 @@ export const Dialog = ({
     previsousOpenedModales,
   } = useModal()
 
+  useEffect(() => {
+    setIsVisible(true)
```

Comment:
this almost like putting the initial value to isVisible to true, so why using a useEffect for this ?

Reply:
There is slight difference, this useEffect content will be run only after the first render. It is important in that case because we need to run an animation and if the first render didn't occur there will be a flickering in the animation as the position are being sets.